src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -721,6 +723,10 @@ def create(
         if "instance_pool_id" not in kwargs:
             kwargs["node_type_id"] = ws.clusters.select_node_type(local_disk=True, min_memory_gb=16)
 
+        if "custom_tags" not in kwargs:
```

Comment:
In line 721 the `kwargs["custom_tags"]` is overwritten, apparently we assume that someone does not supply this field to the `**kwargs` (could be a wrong assumption.

I would prefer if the `{"RemoveAfter": get_test_purge_time()}` is defined once and merged with the `custom_tags` on line 722 or set as `custom_tags` when not `single_node` to avoid another `if-else`